README.adoc

```diff
@@ -183,24 +186,31 @@ The `inventory` service is built in a similar way.
 == Building your container images
 
 A container image is a binary file. It is made up of multiple layers and is used to run code in a container. Images are built from
-instructions in Containerfiles to create a containerized version of the application.
+instructions in `Containerfile` files to create a containerized version of the application.
 
-The syntax for a Containerfile and a Dockerfile are the same. Podman can build your container image using either Containerfiles or Dockerfiles. Containerfiles will be used in this guide.
+The syntax for a `Containerfile` and a `Dockerfile` are the same. 
```

Comment:
subject-verb agreement is wrong here. Should be either:

The syntax is the same for `Containerfile` and  `Dockerfile` files. 
or
`Containerfile` and  `Dockerfile` files use the same syntax.